composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "bitbull/magento2-module-instagramwidget",
   "description": "Instagram stream for Magento2",
   "license": "proprietary",
+  "version": "1.0.1",
```

Comment:
I added because if installing through composer, it needs a version when running `composer update` to pull in latest changes, so should also add version tagging if possible. But can change to whatever version you like.